frontend/src/lib/components/LLMButton/ExplainCSPViolationButton.tsx

```diff
@@ -18,10 +25,43 @@ export const ExplainCSPViolationButton = ({
     const [loading, setLoading] = useState(false)
     const [isOpen, setIsOpen] = useState(false)
     const [result, setResult] = useState<JSX.Element | null>(null)
+    const isMaxEnabled = useFeatureFlag('ARTIFICIAL_HOG')
+
+    const { currentLocation } = useValues(router)
 
     const handleClick = async (): Promise<void> => {
-        setLoading(true)
         setIsOpen(true)
+
+        if (!isMaxEnabled) {
+            setResult(
+                <div className="flex flex-col items-center justify-center min-h-40 gap-4 text-l text-center">
+                    <SupportHeroHog style={{ maxWidth: 120, marginBottom: 8 }} />
+                    <div>
+                        Want to get CSP violation explanations from the cutest security hog in the world?
+                        <br />
+                        <Link
+                            to={
+                                combineUrl(currentLocation.pathname, currentLocation.search, {
+                                    ...currentLocation.hashParams,
+                                    panel: `${SidePanelTab.FeaturePreviews}:${FEATURE_FLAGS.ARTIFICIAL_HOG}`,
+                                }).url
+                            }
+                        >
+                            Enable Max AI
+                        </Link>{' '}
+                        in your feature previews.
+                        <div className="mt-4">
+                            <span className="text-xs text-muted">
+                                Otherwise, you can edit this insight and remove this column on the SQL query.
```

Comment:
I wanted to add a link here to edit the insight directly, but having the `insightLogic` mount here would cause it to explode inside the SQL editor since it is outside the context of insight/dashboard. 

We can probably give it a few more tries, but it seemed reasonable for now.